### PR TITLE
Added models for customers, departments, employees, orders, and produ…

### PR DESCRIPTION
**Pull request by austingorman**
…ct orders

Purpose:
Created models

How it fits in context of the project:
We are creating a few models per person

Specific Feature affected:
Employees, Departments, Customers, Orders, and ProductOrders

How to test (Be thorough!):

Make sure that the models are structured according to the needs defined in the tickets.

### CODE REVIEW

**Comment by Rian501** [comment]
Please elaborate on steps to test. It's not only that the model should exist, but what should we be looking for in it?

**Comment by BShearin99** [comment]
Looks good!

File: Models/Customers.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime DateCreated { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime LastActivity { get; set; }
+
+        List<Customers> CustomerList = new List<Customers>();
```
Comment (Rian501) [inline Models/Customers.cs]: A customer won't actually have a list of customers inside it.
A customer DOES need two lists-- check the customers ticket -- you will need lists for the things that can be included
https://github.com/nashville-software-school/csharp-bangazonapi-boilerplate/issues/1

File: Models/Departments.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
+    public class Departments
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Budget { get; set; }
+        public string Name { get; set; }
+        List<Departments> DepartmentList = new List<Departments>();
```
Comment (Rian501) [inline Models/Departments.cs]: Like Customers, this won't have a list of itself within it. Please review.

**Review by Rian501 — CHANGES_REQUESTED** [review]
Check the tickets for each of these to see what foreign properties need to be included in the model. Also, none of them need a list of themselves.

**Comment by austingorman** [comment]
I think I have the lists fixed

File: Models/Employees.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+namespace BangazonAPI.Models
+{
+    public class Employees
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public Departments DepartmentsId { get; set; }
```
Comment (Rian501) [inline Models/Employees.cs]: Once we get the Department ID we can then get the Dept itself -- like the Customer Customer on the Order.cs class

**Review by Rian501 — COMMENTED** [review]
One small change that I could find.

**Comment by AmandaM04** [comment]
looks good! You have fixed the small issues that I found

**Review by Rian501 — APPROVED** [review]
Looks good to me!

**Pull request merged**